en/docs/chapter_tree/binary_tree_traversal.md

```diff
@@ -22,14 +22,14 @@ Breadth-first traversal is usually implemented with the help of a "queue". The q
 
 ### Complexity analysis
 
-- **Time complexity is $O(n)$**: All nodes are visited once, using $O(n)$ time, where $n$ is the number of nodes.
-- **Space complexity is $O(n)$**: In the worst case, i.e., a full binary tree, before traversing to the lowest level, the queue can contain at most $(n + 1) / 2$ nodes at the same time, occupying $O(n)$ space.
+- **Time complexity is $O(n)$**: All nodes are visited once, taking $O(n)$ time, where $n$ is the number of nodes.
+- **Space complexity is $O(n)$**: In the worst case, i.e., a full binary tree, before traversing to the bottom level, the queue can contain at most $(n + 1) / 2$ nodes simultaneously, occupying $O(n)$ space.
```

Comment:
Maybe **lowest level** will be better

You can check its usage [here](https://echobind.com/post/binary-trees-what-are-they)

Reply:
Thank you, I would keep this one based on [ref](https://www.andrew.cmu.edu/course/15-121/lectures/Trees/trees.html)